.github/workflows/ci.yml

```diff
@@ -17,38 +17,32 @@ jobs:
       with:
         distribution: 'adopt'
         java-version: '11'
-    - name: Coursier cache
-      uses: coursier/cache-action@v6
-    - name: Cache sbt
-      uses: actions/cache@v3
-      with:
-        path: ~/.sbt
-        key: ${{ runner.os }}-sbt-cache-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}-c
+        cache: 'sbt'
+
     - name: Build and test
       run: |
         sbt -v scalafmtSbtCheck scalafmtCheckAll scripted makeSite
-        rm -rf "$HOME/.ivy2/local" || true
-        rm -r $(find $HOME/.sbt/boot -name "*-SNAPSHOT") || true
-        find $HOME/Library/Caches/Coursier/v1        -name "ivydata-*.properties" -delete || true
-        find $HOME/.ivy2/cache                       -name "ivydata-*.properties" -delete || true
-        find $HOME/.cache/coursier/v1                -name "ivydata-*.properties" -delete || true
-        find $HOME/.sbt                              -name "*.lock"               -delete || true
```

Comment:
It wasn't really clear to me why this was needed. Is this actually needed for some reason?

Reply:
Probably historical artifact. If the CI passes it should be ok?

Reply:
> If the CI passes it should be ok?

That's what I thought 😆